include/votca/tools/edge.h

```diff
@@ -89,7 +92,7 @@ template <>
 class hash<votca::tools::Edge> {
```

Comment:
please use just tools:: instead of votca::tools, you are already in namespace votca

Reply:
Yes boss.

Reply:
okay yes true, I am wrong. Sorry